apis/status/v1beta1/constraintpodstatus_types.go

```diff
@@ -53,6 +54,13 @@ type Error struct {
 	Location string `json:"location,omitempty"`
 }
 
+// EnforcementPointStatus represents the status of a single enforcement point.
+type EnforcementPointStatus struct {
+	EnforcementPoint string `json:"enforcementPoint"`
+	Enforced         bool   `json:"enforced"`
```

Comment:
I think it was a mistake to have an `enforced` boolean state with constraints. And with VAP, we cannot make guarantees as to whether they are enforced b/c there is no status reporting, so we cannot know if the VAP object has been ingested or whether there are any errors. Maybe we want something like a string field called `Status`?

There we can put a string like "GENERATED", "ERROR", etc.

Of course, if we call it "status", we should rename the parent field to avoid stutter.

Reply:
how about `code` - `generated/error/...`?

Reply:
"code" seems a bit ambiguous given we are also dealing with "code" in the sense of CEL code.

"state"?

Reply:
I like `state`, updated the name to `state`.